txp/Pos/Script/Examples.hs

```diff
@@ -230,3 +246,10 @@ sigStressRedeemer n = fromE $ parseRedeemer [text|
 
     keyS = sformat fullPublicKeyHexF pk
     sigS = sformat fullSignatureHexF sig
+
+    unwrap (Right a) = a
+    unwrap (Left s) =
+        -- This case is impossible because the input string for 'parseRedeemer'
+        -- is always a valid script regardless of arguments passed to
+        -- 'sigStressRedeemer'.
+        error ("sigStressRedeemer: impossible, failed to parse: " <> s)
```

Comment:
Why did you use `fromE` in some places and `unwrap` in other places?

Reply:
AFAIU, in all places we pass something valid to `parseRedeemer` or `parseValidator`.

Reply:
The reason is that I'm using `liftE` to force compile-time evaluation, so that errors for `parseRedeemer` and `parseValidator` become compile-time errors.

For `multisigRedeemer`, `multisigValidator`, `sigStressRedeemer`, and `shaStressRedeemer` I cannot do that — these `Script`s are parametrized, and the parameters can be specified at run-time. For instance, for `multisigValidator`, the inputs are generated randomly in the test suite.

Reply:
👌